README.md

```diff
@@ -66,10 +66,12 @@ First, the package must be installed:
 Now a prediction can be made for any `qiskit.QuantumCircuit` object or `qasm` file:
 
 ```python
-from mqt.predictor.driver import compile
+from mqt.predictor import ml, rl
 
-compiled_qc_ML, predicted_best_device_ML = compile("qasm_file_path", model="ML")
-compiled_qc_RL, predicted_best_device_RL = compile(
+compiled_qc_ML, compilation_info_ML = ml.compile_prediction(
+    "qasm_file_path", model="ML"
+)
+compiled_qc_RL, compilation_info_RL = rl.compile_prediction(
     "qasm_file_path", model="RL", opt_objective="fidelity"
 )
```

Comment:
Love the simplified interface here.
Depending on whether you decide to adopt the changes down below, don't forget to update here as well.

Reply:
Adjusted it here as well.